packages/web/app/src/components/ui/tabs.tsx

```diff
@@ -37,6 +71,18 @@ const TabsTrigger = React.forwardRef<
 ));
 TabsTrigger.displayName = TabsPrimitive.Trigger.displayName;
 
+const TabsContentMenu = React.forwardRef<
+  React.ElementRef<typeof TabsPrimitive.Content>,
+  React.ComponentPropsWithoutRef<typeof TabsPrimitive.Content> & { noPadding?: boolean }
+>(({ className, noPadding, ...props }, ref) => (
+  <TabsPrimitive.Content
+    ref={ref}
+    className={cn(noPadding ? undefined : 'py-7', className)}
```

Comment:
```suggestion
    className={cn(!noPadding && 'py-7', className)}
```

Reply:
personally, I don't like I don't like `props` aka `noPadding`, `noMargin`, you have already `className` and padding can be removed with `p-0` cc @kamilkisiela @n1ru4l